### PR TITLE
Tabs

### LINKED ISSUE #26: TabView Tabs
### Description
Defining individual tabs for a tab view doesn't seem to be implemented. I'd like to create a PR to implement this and am wondering what the API might look like.

I was thinking of something like this:

```fs
View.tabView [
  tabView.tabs [
    tabView.tab [
      tab.title "Tab 1"
      tab.view [
        View.label [ label.text "Hello, World!" ]
      ]
    ]
    tabView.tab [
      tab.title "Tab 2"
      tab.view [
        View.label [ label.text "Hello, World!" ]
      ]
    ]
  ]
]
```

I see that `prop.ref` exists to reference the TabView element directly and add tabs that way but I think a feliz-style definition would be quite nice to add.

### PR DESCRIPTION
**Pull request by sheridanchris**
Implement a feliz-style DSL for creating tabs. This will close #26

### CODE REVIEW

**Comment by sheridanchris** [comment]
I added a tab view example but it doesn't seem to be working. With some logging I can see that a tab is getting added, but the tab view doesn't render. I'll have look into this some more.

**Comment by sheridanchris** [comment]
Everything seems to be working properly.

File: flake.nix
Comment (Hardt-Coded) [inline flake.nix:1]: Do we need this flake files?

Comment (sheridanchris) [reply flake.nix:1]: Not really. I need them as I use flake's to create reproducible development environments. They can be deleted.

**Pull request merged**